Sources_v3/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -0,0 +1,203 @@
+//
+// Copyright © 2020 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import UIKit
+
+open class ChatChannelListItemView<ExtraData: UIExtraDataTypes>: UIView, AppearanceSetting {
+    // MARK: - Default Appearance
+    
+    public class func initialAppearanceSetup(_ view: ChatChannelListItemView<ExtraData>) {
+        if #available(iOS 13, *) {
+            view.backgroundColor = .systemBackground
+        } else {
+            view.backgroundColor = .white // Should be add custom support for dark theme?
+        }
+    }
+    
+    // MARK: - Properties
+    
+    public let uiConfig: UIConfig<ExtraData>
+    
+    public var channel: _ChatChannel<ExtraData>? {
+        didSet {
+            updateContent()
+        }
+    }
+    
+    // MARK: - Subviews
+    
+    public private(set) lazy var container = ContainerStackView().withoutAutoresizingMaskConstraints
+    
+    public private(set) lazy var avatarView: ChatChannelAvatarView<ExtraData> = {
+        uiConfig.channelList.channelListItemUI.avatarView.init().withoutAutoresizingMaskConstraints
+    }()
+    
+    public private(set) lazy var titleLabel = UILabel().withoutAutoresizingMaskConstraints
+    
+    public private(set) lazy var subtitleLabel = UILabel().withoutAutoresizingMaskConstraints
+    
+    public private(set) lazy var unreadCountBubble: ChatUnreadCountBubbleView = {
+        uiConfig.channelList.channelListItemUI.unreadCountView.init().withoutAutoresizingMaskConstraints
+    }()
+    
+    public private(set) lazy var readStatusView: ChatReadStatusCheckmarkView = {
+        uiConfig.channelList.channelListItemUI.readStatusView.init().withoutAutoresizingMaskConstraints
+    }()
+    
+    public private(set) lazy var timestampLabel = UILabel().withoutAutoresizingMaskConstraints
```

Comment:
The property wrapper looks/reads better IMO, especially with the long lines.

```swift
public let label = UILabel().withoutAutoresizingMaskConstraints

@UseAutoLayout
public let label = UILabel()
```

Reply:
Sorry, but wrapping a view into a custom property is too much 😅 Also, it won't work with lazy views, correct?

Reply:
Setting this bool flag for views is a non-issue if we _always_ use our layout helpers, which will set the flag, like SnapKit does.